### PR TITLE
Homepage CSS Updates

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds Grunt, LESS, etc.
- Centers items in main navbar

Before:
Screen Shot 2020-02-20 at 3 43 56 PM

After:
Screen Shot 2020-02-20 at 3 43 45 PM

## Why

For #3

## Testing/Questions

Features that this PR affects:

- Main navbar

Questions that need to be answered before merging:

- [ ] Is this PR targeting the correct branch in this repository?

Steps to test this PR:

1. View the main navbar and make sure it looks ok

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -119,3 +119,4 @@ wp-content/debug.log
 # add a line here that looks like this:
 # !/wp-content/themes/sirf
 !/wp-content/themes/sirf
+wp-content/themes/sirf/node_modules/*
```
Comment (benlk) [inline .gitignore]: We should probably add this to the `.gitignore` in the sirf theme itself.

File: wp-content/themes/sirf/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "sirf-theme",
+  "version": "1.0.0",
+  "description": "Theme for sirf-onlien.org",
```
Comment (benlk) [inline wp-content/themes/sirf/package.json]: typoed 'online' but that's low priority to fix.

Comment (joshdarby) [reply wp-content/themes/sirf/package.json]: Fixed in https://github.com/INN/umbrella-sirf/pull/7/commits/5430fc9002c9cecf65ead323e69437ef9e58eecc

File: .gitignore (hunk 1 above)
Comment (joshdarby) [reply .gitignore]: Moved in https://github.com/INN/umbrella-sirf/pull/7/commits/5430fc9002c9cecf65ead323e69437ef9e58eecc

**Pull request merged**